src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/TypeLoaderEnvironment.cs

```diff
@@ -119,6 +119,8 @@ internal static void Initialize()
             Instance = new TypeLoaderEnvironment();
             RuntimeAugments.InitializeLookups(new Callbacks());
             s_nativeFormatStrings = new LowLevelDictionary<string, IntPtr>();
+            s_runtimeFieldHandles = new LowLevelDictionary<RuntimeFieldHandleKey, RuntimeFieldHandle>();
```

Comment:
Great :). I actually spent some time trying to find out this "no reason" answer, but wasn't 100% sure if it was really "no reason" or there was a hidden reason I did not know about.

Thanks for confirming. I'll convert these to instance variables.